spec/spec_helper.rb

```diff
@@ -41,7 +41,7 @@
 
 
 # Handy constants for initializing PG connections:
-QUE_URL = ENV['DATABASE_URL'] || 'postgres://postgres:@localhost/que-test'
+QUE_URL = ENV['DATABASE_URL'] || 'postgres://localhost/que-test'
```

Comment:
Hm, this has been changed and then changed back on two separate occasions just within this PR. Suspect we need to get to the bottom of what the correct behaviour is. I made 0614904c67a5a982e84aaeec3491803cba04144c  because the earlier change wasn’t working for me.

Reply:
Whatever we do for this, we should also update this in [`tasks/safe_shutdown.rb`](https://github.com/greensync/que/blob/5c5d3bccb2ef515fcfa1c5def47bf9571ae62f3a/tasks/safe_shutdown.rb#L22)

Reply:
I've rebased to remove all changes to this now that we have https://github.com/que-rb/que/pull/324